packages/metro-config/src/ExpoMetroConfig.ts

```diff
@@ -1,4 +1,4 @@
-import { getDefaultTarget, ProjectTarget } from '@expo/config';
+import { getConfig, getDefaultTarget, isLegacyImportsEnabled, ProjectTarget } from '@expo/config';
```

Comment:
darn, more @expo/config deps

Reply:
Better than xdl :| we should stay on deprecating all of the new features as soon as we can.